test/chi0.jl

```diff
@@ -44,7 +44,7 @@ include("testcases.jl")
                                        use_symmetry=use_symmetry)
                 energies, ham = energy_hamiltonian(basis, nothing, nothing; ρ=ρ0)
                 ρ2 = DFTK.next_density(ham, tol=tol, eigensolver=diag_full, n_bands=n_bands).ρ
-                diff_findiff = (ρ2 - ρ1) / ε
+                diff_findiff = from_real(basis, ρ2.real - ρ1.real) / ε
```

Comment:
Why? This should work otherwise as well if we implemented things properly.

Reply:
It's failing the assert that both are on the same basis. I guess we could remove the assert?

Reply:
yeah let's trust the user on this one

Reply:
Ah ok. That I did not have in mind. No I'd rather keep the assert. Leave it as is.

Reply:
Too late I removed it :-p I think it's perfectly legitimate to want to eg compare densities between two systems with different atomic positions

Reply:
all right, whatever, not sure I care that much